content/nap-waf/v5/admin-guide/install.md

```diff
@@ -590,7 +590,7 @@ Follow these steps before you install NGINX Open Source or NGINX Plus.
     ```
 
 {{%/tab%}}
-{{%tab name="Ubuntu 20.04"%}}
+{{%tab name="Ubuntu 22.04"%}}
```

Comment:
Actually, we already have a tab for Ubuntu 22.04. So the solution here is to delete lines 593-606. (Same issue, 3 places total)